lib/decimal/binary-to-decimal.cc

```diff
@@ -312,6 +312,15 @@ void BigRadixFloatingPointNumber<PREC,
   }
 }
 
+#define INSTANTIATE_WALDO(X) \
```

Comment:
What's this, here and in `decimal-to-binary.cc`?

Reply:
These are hacks to get these files to build correctly on my Mac.

Reply:
What was broken?  Were you getting unresolved external names?

Reply:
It was a long time ago. But as far as I remember, yes, the builds started failing after this code was introduced with link-time failures.

Reply:
This might be a work-around to a compiler bug with template instantiation in a specific environment.  Please confirm that the bug persists; if it does, other work-arounds can be tried.

Reply:
It appears to have been a transitory issue. This code will be removed.